### DISCUSSION TITLE
Symbols Size

### DISCUSSION THREAD

**Discussion by ghost** [JanDeDobbeleer/oh-my-posh#1661] [Troubleshoot]
## About my question
My question is about the ***leading_diamond***  propertie.
![image](https://user-images.githubusercontent.com/70576040/151322341-899fa5f9-ebfe-4430-a2da-8c6fabdd9bb0.png)
 Is there a way to resize the symbol so that it appears larger?
![image](https://user-images.githubusercontent.com/70576040/151322597-4de321fc-d1cb-409b-a74a-fee0947f4a0f.png)
On my console, the icon looks so small...

**Comment by JanDeDobbeleer** [comment]
No, that's related to the console (settings) or font. We can't influence that in any way.
